src/Nethermind/Nethermind.State/IWorldState.cs

```diff
@@ -108,4 +108,11 @@ public interface IWorldState : IJournal<Snapshot>, IReadOnlyStateProvider
 
     void CommitTree(long blockNumber);
     ArrayPoolList<AddressAsKey>? GetAccountChanges();
+
+    /// <summary>
+    /// PreBlockCache related
+    /// </summary>
+    bool SupportPreBlockCache => false;
+    bool IsCacheDirty => false;
+    void ClearCache() { }
```

Comment:
Can we at least hide it under some `PreBlockCache` property?

Reply:
Are you sure? `PreBlockCaches` is clearly implementation specific. I'd even err towards `PrepareForPreBlockWarmer()` instead of the `IsCacheDirty` and `ClearCache` here.

Reply:
We can expose something that has only `IsCacheDirty` and `ClearCache`. I just want to minimize elements in `IWorldState`